packages/plugin-dom/DomRenderer.ts

```diff
@@ -122,54 +104,16 @@ export class DomRenderer extends Renderer<
             renderedDomNodes.push(placeholderBr);
         }
 
-        currentContext.parentNode.appendChild(fragment);
-
-        return new Map(
-            renderedDomNodes.map(domNode => {
-                return [domNode, [node]];
-            }),
-        );
-    }
-    /**
-     * Return the next rendering context, based on the given context. This
-     * includes the next VNode to render and the next parent element to render
-     * it into.
-     *
-     * @param currentContext
-     */
-    _nextRenderingContext(currentContext: DomRenderingContext): DomRenderingContext {
-        const node = currentContext.currentNode;
-        if (node.hasChildren()) {
-            currentContext.currentNode = node.firstChild();
-            // Render the first child with the current node as parent.
-            const renderedParent = VDocumentMap.toDom(node);
-            if (renderedParent) {
-                currentContext.parentNode = renderedParent;
-            }
-            this._contextStack.push({ ...currentContext });
-        } else if (node.nextSibling()) {
-            // Render the siblings of the current node with the same parent.
-            this._contextStack[this._contextStack.length - 1].currentNode = node.nextSibling();
-        } else {
-            // Render the next ancestor sibling in the ancestor tree.
-            let ancestor = node;
-            // Climb up the ancestor tree to the first parent having a sibling.
-            while (ancestor && !ancestor.nextSibling() && ancestor != currentContext.root) {
-                ancestor = ancestor.parent;
-                this._contextStack.pop();
+        // Render the node's children
+        let rendered = [];
+        node.children.forEach(child => {
+            if (!rendered.includes(child)) {
+                rendered = [...rendered, ...this.renderNode(child, renderedElement).keys()];
```

Comment:
Yeah, really that's not ok :p Actually, with this mechanism, is the map still needed ? I think it is yes, but we said we would set it manually. That's something else though.